packages/bpk-component-modal/src/BpkModalInner.tsx

```diff
@@ -125,6 +127,7 @@ const BpkModalInner = ({
         tabIndex={-1}
         role="dialog"
         aria-labelledby={showHeader ? headingId : undefined}
+        aria-label={!showHeader ? ariaLabel : undefined}
```

Comment:
Could skip this guard but seems sensible to protect the consumer from providing a title and aria label